### PR TITLE
Members photos reorganization

### PR DESCRIPTION
**Pull request by DavidBronfen**
#430

### CODE REVIEW

**Comment by amitaibu** [comment]
@DavidBronfen the images seems to be really big, can you go over them please and resize

**Comment by DavidBronfen** [comment]
@amitaibu 
All the images has been reduced to fit the needed size for the website.

File: _config.yml [hunk 1]
```diff
@@ -75,24 +75,16 @@ authors:
       github: OritGeron
       avatar: OritGeron.jpg
       image: OritGeron.jpg
-  - HelenaEksler:
```
Comment (amitaibu) [inline _config.yml]: why remove? Everybody should stay, only `is_member` should be set to `false`

File: _config.yml [hunk 2]
```diff
@@ -221,30 +203,86 @@ authors:
...
+      github: DavidHernandez
+      avatar: DavidHernandez.jpg
       image: male.jpg
   - itamarbu:
       name: Itamar Burstein
       title: Industrial Designer
       is_member: false
       bio: My designs have been endorsed by companies such as VIA France, Roche Bobois, Metalarte, Ligne Roset & Jofebar. That's pretty awesome.
       avatar: itamarbu.jpg
-  - ceoaliongroo:
```
Comment (amitaibu) [inline _config.yml]: keep everybody (you can search and see they have posts - that will be broken if you remove this)

**Comment by DavidBronfen** [comment]
@amitaibu 
All the missed data has been reverted (Helena, Yaron and Carlos).

**Comment by amitaibu** [comment]
@DavidBronfen ok, you have to run this on your local, to see why it's not ready. It's kind of funny even :)

**Comment by DavidBronfen** [comment]
@amitaibu 
Looks nice :)
Roy tried to help me yesterday (without success) with the jekyll issue on my local.. 
Do you think you'll have a free moment to take a look on it?

**Comment by amitaibu** [comment]
Needs a re-roll

**Comment by DavidKohav** [comment]
@amitaibu 
Done.

**Comment by amitaibu** [comment]
Savyon's avatar was lost, I had to re-add it.

@kazabooboo you are the only one currently missing an image - could you please provide one.

**Comment by kazabooboo** [comment]
@amitaibu sent one some time ago. Link to hi res https://drive.google.com/open?id=0B30gFjtQyH8_WnlCdVAtaGJVa0k

**Comment by amitaibu** [comment]
@kazabooboo , great - can you open a new issue, and assign to @DavidBronfen

**Comment by kazabooboo** [comment]
@amitaibu done #499

**Pull request merged**